src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs

```diff
@@ -31,7 +32,7 @@ public class MongoDbJournal : AsyncWriteJournal
         private static readonly BsonTimestamp ZeroTimestamp = new BsonTimestamp(0);
         
         private readonly MongoDbJournalSettings _settings;
-
+        private MongoClient _client;
```

Comment:
Can you do the same with this as the database in the snapshot class?
You can get a reference to the `IMongoClient` by using `_mongoDatabase.Value.Client`